test/integration/datapath/datapath_windows_test.go

```diff
@@ -48,12 +53,35 @@ Timeout context is controled by the -timeout flag.
 func setupWindowsEnvironment(t *testing.T) {
 	ctx := context.Background()
 
+	t.Log("Get REST config")
+	restConfig := kubernetes.MustGetRestConfig(t)
+
 	t.Log("Create Clientset")
 	clientset, err := kubernetes.MustGetClientset()
 	if err != nil {
 		t.Fatal(err)
 	}
 
+	if *initBYOCNI {
+		// create ipmasq configmaps
```

Comment:
We'll need to remove the yaml deployments from the pipeline yaml i suppose.

Reply:
Actually, removing this new code that isntalls, and leaving installation code in pipeline
If we really want to later, I can remove it all pipelines and make it only in go code in "InstallCNSDaemonset" method